web_ckeditor4/static/src/js/web_ckeditor4.js

```diff
@@ -122,8 +122,11 @@ openerp.web_ckeditor4 = function(instance)
             removePlugins: 'iframe,flash,forms,smiley,pagebreak,stylescombo',
             filebrowserImageUploadUrl: 'dummy',
             extraPlugins: 'filebrowser',
-            // this is '#39' per default which screws up single quoted text in ${}
-            entities_additional: '',
+            protectedSource: [
+                /\n*\s*%.*\n*/gm, // line expressions, keep newline if any
```

Comment:
@hbrunn I don't know why but this comments aren't minifing correctly and broke JS, check it in runbot.